pygmt/src/text.py

```diff
@@ -76,8 +76,22 @@ def text_(  # noqa: PLR0912
     Parameters
     ----------
     textfiles : str or list
-        A text data file name, or a list of file names containing 1 or more
-        records with (x, y[, angle, font, justify], text).
+        A file name or a list of file names containing one or more records.
+        Each record has following columns:
+
+        * *x* : X coordinate or longitude
+        * *y* : Y coordinate or latitude
+        * *angle* : Angle in degrees counter-clockwise from horizontal
+        * *font* : Text font
```

Comment:
Maybe we should mention that `font` allows not only to set the font itself but also the size and color?